WTWSMS/decisions/minor_decisions.txt

```diff
@@ -3,11 +3,6 @@ decisions = {
 	feast_0 = {
 		potential = {
 			is_ruler = yes
-			OR = {
-				is_feudal = yes
-				is_tribal = yes
-				is_nomadic = yes
-			}
 			prisoner = no
 			OR = {
 				religion_group = christian
```

Comment:
Why can't any government meeting the religious requirements hold a feast? Are you telling me that wealthy Merchant Republics didn't pig out from time to time?